content/rules.md

```diff
@@ -0,0 +1,105 @@
+- Changing the composition of a team is possible after approval by the organization.
+- Changing the composition should be done before the start of the contest the substitutions should be registered as a reserve.
+- The organization decides the amount of teams for each institution allowed competing.
+- The responsibility for listing which of its teams+reserves should be considered for participation at the BAPC lies with their institute.
+- The organization has the right to deny the participation of teams before the start of the contest.
+
+### Photos and videos
+
+There might be a public livestream of the award ceremony available on the website. The recording will be publicly available afterwards too. If you do not want to appear on this livestream or want to be removed from the footage afterwards, please contact the organization. The organization can decide to not grant this request.
+
+### Student teams
+
+A student team:
+
+- Participates for free.
+- Exists of students from the same institution and who are not participating in another team. The institutions must be located in either Belgium, Luxembourg or The Netherlands.
+- Has a coach, which is the contact person of a team. This should be a student or faculty member of the institution.
+- Participates in the student teams pool for the title "Winner of the Benelux Algorithm Programming Contest 2021".
+- Consists of students who are eligible for the North Western European Programming Contest 2021.
+
+### Business teams
+
+A business team:
+
+- Consists up to 3 persons who are employed by the same company or institution.
+- Participates in the business teams pool for the title "Winner of the Benelux Algorithm Programming Contest 2021".
+
+## The Contest
+
+### Introduction
+
+- The language of the contest is English. All written contest materials will be in English.
+- The contest lasts for 5 hours.
+- From the beginning until one hour before the end of the BAPC, the scores are displayed.
+
+### Problems
+
+- At least six problems will be posed. The jury intents to develop problems with as little as possible dependence on detailed knowledge of a particular applications area or a particular contest language.
+- A contestant may submit a claim of ambiguity or error in a problem statement by submitting a clarification request to a judge. If the judges agree that an ambiguity or error exists, a clarification will be issued to all contestants.
+- The jury has the right to change or withdraw problems during the contest before informing all teams.
+
+### System
+
+- A solution for a problem has to be written in C99, C++ (version 14 or later will be available), Python (versions 3 will be available), Java (version 11 or later) or Kotlin (version 1.3), unless the problem statement explicitly states otherwise. Business teams can request other submission languages, on which the organization will decide whether it will be made available for submission.
```

Comment:
I think the Java version should be 8+...

Reply:
I think I directly copied this section and should look at it again with real values. Might be smart to discuss this soon with the other sides.

Reply:
When we build the image we will revisit this.